src/styles/index.css

```diff
@@ -17,8 +21,12 @@ body {
   height: 100vh;
 }
 
-* {
+html {
   font-size: 16px;
```

Comment:
distinguished the html font size (16px) from the default font size (14px) because the `rem` unit is calculated using the font size of the html element. tailwind classes are primarily defined using `rem`, so we need to maintain rem of 16px.